src/Google/RequestReviewStatuses.php

```diff
@@ -128,7 +128,7 @@ private function maybe_update_status( $new_status, $status ) {
 	private function maybe_load_eligible_region( array $region_status, array $review_eligible_regions ) {
 		if (
 			isset( $region_status['regionCodes'] ) &&
-			count( $region_status['regionCodes'] ) &&
+			! empty( $region_status['regionCodes'] ) &&
```

Comment:
💅  While we're using `empty` now I think `isset` could be removed.

Reply:
Til